drivers/nss/lab01/ex1/hello.c

```diff
@@ -0,0 +1,22 @@
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/kernel.h>
+
+/* TODO 1: create the initialization and cleanup (exit) functions */
+/* TODO 2: add module information */
+
```

Comment:
Are these TODO1 and TODO2 supposed to be here? I think we should remove them because there are already next TODOs that say to implement init/cleanup function.

Reply:
done